cupboard_app/views.py

```diff
@@ -784,3 +791,83 @@ def list(self, request: Request) -> Response:
         all_measurements = get_all_measurements()
         serializer = MeasurementSerializer(all_measurements, many=True)
         return Response(serializer.data, status=200)
+        return Response(serializer.data)
+
+
+@extend_schema(tags=['CustomIngredients'])
+class CustomIngredientsViewSet(viewsets.ViewSet):
+    MISSING_ING = 'Missing ingredient and type in message body.'
+
+    @extend_schema(
+        request=None,
+        responses={
+            201: CustomIngredientSerializer,
+            400: MessageSerializer,
+            401: auth_failed_response,
+        },
+        examples=[
+            OpenApiExample(
+                name='Custom Ingredient Created',
+                value={'name': 'Beef', 'type': 'Meat'},
```

Comment:
Could you change value to equal SINGLE_CUSTOM_INGREDIENT_DICT

Also add response_only=True flag to the other two examples.